### PR TITLE
feat(shared): migrate alert() calls to Toast notifications

### PR DESCRIPTION
**Pull request by tunile943**
## Summary by CodeRabbit

* **New Features**
  * Replaced browser alerts with in-app toast notifications for meeting creation, editing, deletion, administrative actions, exports, and sync operations.
  * Added clearer success and error feedback while preserving detailed server-provided error messages.

* **Tests**
  * Updated end-to-end coverage to validate toast notifications and in-page confirmation dialogs instead of native browser dialogs.

## Description
- **frontend/app/(main)/page.tsx**: 4 sites — delete success/error, suspend error, resume error.
- **frontend/app/components/meeting-form/NewMeeting.tsx**, **EditMeeting.tsx**: create/update success alerts → toasts. (The last remaining `alert()` in each — client-side validation — is deliberately left for the stacked PR that replaces it with a dedicated banner.)
- **frontend/app/components/admin/diagnostics/SuspendedCard.tsx**, **SyncIssuesCard.tsx**: resume/retry-sync error alerts → toasts.
- **frontend/app/components/admin/users/UsersTab.tsx**: 6 sites — invite/role-change/remove, including the existing HTTP-409 race-condition message preserved as-is.
- **frontend/app/components/admin/export/ExportTab.tsx**: 3 sites — save-settings/save-export-fields errors.
- **frontend/tests/e2e/{02-meeting-creation,03-meeting-editing,04-meeting-deletion,06-zoom-integration,07-google-calendar-sync,09-edge-cases,10-recurring-meetings,17-mobile-meeting-forms}.spec.ts**: removed native `page.on/once/waitForEvent("dialog", ...)` handling made obsolete now that these alerts are toasts, replacing message-content assertions with toast-visibility checks where one previously existed. Two of these (10-recurring-meetings) were actually latent bugs — `waitForEvent("dialog")` would have hung indefinitely once the alert stopped firing, not just silently no-op'd.

Every site's existing message/logic is preserved exactly — only the delivery mechanism changed from `alert()` to `showToast()`. Two stale "please check the Meeting collection on MongoDB" references (left over from the pre-Postgres-migration era) were dropped from the create/update success messages while touching those lines anyway.

## Testing
- [x] `yarn test:all` green (92 e2e + lint/unit/component/integration)
- [x] Grepped the full e2e suite for any other `dialog`-handling tied to the migrated alerts before considering this done (per this repo's test-coverage convention) — none missed.

## Area(s) Touched
**Product areas**
- [x] admin — /admin shell (Diagnostics, Users, Import, Export tabs)
- [x] ui/ux — frontend layout/styling not tied to a specific integration

**Engineering**
- [x] testing — test suite (Jest/Playwright) changes

## Pre-merge Checklist
- [x] Code follows current formatting conventions and passes lint (`yarn lint`).
- [x] Comments are appropriate — only where genuinely non-obvious, not restating what the code already says.
- [x] All test suites pass, both run locally and green in GitHub CI. **Do not merge if any test suite is failing.**
- [x] A test suite was added or updated for the feature/fix in this PR. **Double-check this if you change a feature and did not update the test suites**.
- [x] Only files relevant to this change are committed — no stray or unrelated files.
- [x] If this branch has a merge conflict with master, master was merged into this branch first (not the other way around).
- [x] The rest of this PR description (Description, Testing) is filled out, not left as template placeholders.

### CODE REVIEW

**Comment by tunile943** [comment]
@coderabbitai rate limit

**Comment by tunile943** [comment]
@coderabbitai review

File: frontend/app/(main)/page.tsx [hunk 1]
```diff
@@ -19,8 +19,10 @@ import { useConflictMids } from "../../hooks/useConflictMids";
 import { useViewport } from "../../hooks/useViewport";
 import { PHONE_BREAKPOINT } from "../../util/common/breakpoints";
 import { useCalendarContext } from "../context/CalendarProvider";
+import { useToast } from "../components/shared/ToastProvider";
 
 export default function HomePage() {
+  const { showToast } = useToast();
```
[the remark this answers is not in the document]
Comment (tunile943) [reply frontend/app/(main)/page.tsx:25]: Fixed — but the actual line lives in `Toast.tsx`, which is owned by the base branch `feat/toast-core` (#352), not this PR's diff. Applied `role="alert"`/`aria-live="assertive"` for the error variant there (commit 5fac892) and cascaded the rebase up through the whole stack.

**Review by icr-dev — DISMISSED** [review]

**Review by icr-dev — APPROVED** [review]
